deploy/olm-catalog/manifests/assisted-service-operator.clusterserviceversion.yaml

```diff
@@ -664,18 +664,22 @@ spec:
           - create
         - apiGroups:
           - ""
+          - coordination.k8s.io
```

Comment:
aren't those generated? https://coreos.slack.com/archives/G01A5NB3S6M/p1642489754024700

Reply:
this change is the product of running generate-all. But following the discussion I am adding also the operator changes